### PR TITLE
Bugfix in water cost equation 

### PR DESCRIPTION
**Pull request by vartika271987**
## :bird: Description of this PR :bird:

- This PR corrects a bug in the water cost equation. This affects only India runs at present, and only in the case when pumping cost settings are used for India. No change in the MAgPIE default run. 

## :wrench: Checklist for PR creator :wrench:

- [ ] Label pull request [from the label list](https://github.com/magpiemodel/magpie/labels).
  - **Low risk**: Simple bugfixes (missing files, updated documentation, typos) or changes  in start or output scripts

- [ ] Self-review own code
  - No hard coded numbers and cluster/country/region names.
  - The new code doesn't contain declared but unused parameters or variables.
  - [`magpie4`](https://github.com/pik-piam/magpie4) R library has been updated accordingly and backwards compatible where necessary.
  - `scenario_config.csv` has been updated accordingly (important if `default.cfg` has been updated)

- [ ] Document changes 
  - Add changes to `CHANGELOG.md`

- [ ] Perform test runs
  - **Low risk**: 
    - Run a compilation check via `Rscript start.R --> "compilation check"`
  - **Medium risk**: 
    - Run test runs via `Rscript start.R --> "test runs"`
    - Check logs for errors/warnings

## :rotating_light: Checklist for reviewer :rotating_light:

- PR is labeled correctly
- Code changes look reasonable
  - No hard coded numbers and cluster/country/region names.
  - No unnecessary increase in module interfaces
  - model behavior/performance is satisfactory.
- Changes are properly documented
  - `CHANGELOG` is updated correctly
  - Updates in interfaces have been properly addressed in the module documentations
  - In-code documentation looks appropriate
- [ ] content review done (at least 1)
- [ ] RSE review done (at least 1)

### CODE REVIEW

File: CHANGELOG.md
Comment (FelicitasBeier) [inline CHANGELOG.md:1]: For some reason there are more differences in the changelog than the line that you have added.
Often just a doubling of the headline (e.g. Unreleased Unreleased). Maybe a merge conflict thing? 
Could you clean that up?

File: main.gms [hunk 1]
```diff
@@ -150,20 +150,19 @@ $title magpie
 * 
 * Used data set: rev4.109_h12_magpie.tgz
 * md5sum: NA
-* Repository: scp://cluster.pik-potsdam.de/p/projects/rd3mod/inputdata/output
+* Repository: https://rse.pik-potsdam.de/data/magpie/public
```
Comment (FelicitasBeier) [inline main.gms:153]: Here I don't know which one is better to have in the main.gms.
@tscheypidi

**Review by FelicitasBeier — CHANGES_REQUESTED** [review]

File: main.gms (hunk 1 above)
Comment (tscheypidi) [reply main.gms:153]: does not matter much, but if I have to choose then I would go with the rse.pik-potsdam.de address as this one is publicly available

File: CHANGELOG.md
Comment (vartika271987) [reply CHANGELOG.md:1]: I tried to fix it after a version was released yesterday, but it seems it has changed a lot of things, its not clear to me. Can you please check if it looks okay now? 
Or I will drop this one and create a new PR altogether.

**Review by FelicitasBeier — APPROVED** [review]

**Review by k4rst3ns — COMMENTED** [review]
Looks good to me.
I guess this is not critical, as water pumping cost were not active for most cases, is that correct?
Just out of curiosity, I would be interested in the difference that makes for irrigation dynamics in the model ^^'

**Review by k4rst3ns — APPROVED** [review]

**Comment by vartika271987** [comment]
> Looks good to me. I guess this is not critical, as water pumping cost were not active for most cases, is that correct? Just out of curiosity, I would be interested in the difference that makes for irrigation dynamics in the model ^^'

- Water pumping costs were not active by default, and even when activated only contain values for India. So, only India runs were affected.

**Pull request merged**